languages/javascript/exercises/concept/array-destructuring/array-destructuring.js

```diff
@@ -0,0 +1,21 @@
+export function getFirstCard(deck) {
```

Comment:
```suggestion
@ts-check

export function getFirstCard(deck) {
```

([`@ts-check`](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html#ts-check))